app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,80 @@
  *= require_tree .
  *= require_self
  */
+body {
+  background-color: #3B5999;
+  text-align: center;
+}
+
+li {
+  display: inline-block;
+  padding-right: 6px;
+  padding-left: 6px;
+}
+
+ul {
+  padding-top: 24px;
+  padding-bottom: 24px;
+  padding-right: 6px;
+  padding-left: 6px;
+}
+
+h3{
+  margin-top: 8px;
+  margin-bottom: 8px;
+}
+
+.sign-up-in-tile {
+  font-family: 'Ubuntu', sans-serif;
+	overflow: auto;
+	box-sizing: border-box;
+  background-color: white;
+  /* background: #B9A583 none repeat scroll 0% 0% !important; */
+	width: 800px;
+	height: 400px;
+	border-radius: 30px !important;
+	box-shadow: rgba(0, 0, 0, 55) 0px 16px 30px !important;
+	position: absolute;
+	top: 50%;
+  left: 50%;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+  padding: 24px;
+}
+
+.post-text {
+  font-family: 'Ubuntu', sans-serif;
+  padding: 8px;
+  border-bottom: 2px solid grey;
+  border-right: 1px solid grey;
+  background: white;
+  border-radius: 0.15rem;
+  margin-bottom: 10px;
+  text-align: left;
+}
+
+.post-date {
+  font-size: 12px;
+  font-family: "Courier New";
+  color: darkgrey;
+  text-align: right;
+}
+
+.button_to {
+  display: inline-block;
+}
+
+textarea {
+  border: none;
+  resize: none;
+  border-radius: 0.35rem;
+  font-family: 'Ubuntu', sans-serif;
+  border-bottom: 2px solid grey;
+  border-right: 1px solid grey;
+  padding-left: 10px;
+  padding-top: 8px;
+}
+
+#new_post {
```

Comment:
Is this affecting anything? And could it be turned into a class as opposed to an id?

Reply:
It is an auto-generated id name for the form for creating new posts. It has also a class name of new_post. I think it makes little difference if we use the class name instead of the id name